script.js

```diff
@@ -130,6 +130,18 @@ function renderGameOver() {
   document.querySelector('.js-options-list').classList.add('is-hidden');
   document.querySelector('.js-quiz-question').classList.add('is-hidden');
   document.querySelector('.js-game-over').classList.remove('is-hidden');
+  document.querySelector('.game-restart-button').addEventListener('click', (event) => {
+    restartGame(event);
+  });
+}
+
+function restartGame() {
+  playGame();
+  store.score = 0;
```

Comment:
make a function `resetScore` which sets score to 0, and call that function here. The func shouldn't accept any params.